packages/paste-website/src/pages/libraries/codemods/index.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Codemods
+description: A collection of jscodeshift codemods to help update projects built with Paste.
+slug: /libraries/codemods/
+---
+
+import {graphql} from 'gatsby';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+
+export const pageQuery = graphql`
+  {
+    allPasteLibraries(filter: {name: {eq: "@twilio-paste/codemods"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/libraries/codemods/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Codemods"
+  categoryRoute={SidebarCategoryRoutes.LIBRARIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-codemods"
+  data={props.data.allPasteLibraries.edges}
+/>
+
+---
+
+<contentwrapper>
+<PageAside data={props.data.mdx} />
+<content>
+
+## About
+
+This library provides codemods to alleviate the burden of teams making large sweeping changes related to Paste.
```

Comment:
What are codemods? Is there a page or something we can link someone to for an explainer or from where they came from?

Reply:
https://github.com/facebook/jscodeshift
https://github.com/reactjs/react-codemod

```
This library is a collection of scripts that update your codebase from old patterns to new patterns automagically. In this way we can effectively move API's and patterns to newer better versions without incurring tech debt across the entire Twilio organization.

Notable users of this approach are React, Ember and Vue. All of those frameworks provide assistance to downstream users when they introduce breaking changes to alleviate pain points and provide a path forward for large codebases.
```

That's my stab at wordage but I am not a great wordsmith